StepDefinitions/LanguageStep.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Hemz_QA_Mars.Pages;
+using TechTalk.SpecFlow;
+
+namespace Hemz_QA_Mars.StepDefinitions
+{
+    [Binding]
+    public class LanguageStep
+    {
+        Languages Language_Obj = new Languages();
```

Comment:
make use of constructors 
Please see 
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

Reply:
Yes, I have used constructor now (refer below) and updated in all the places
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/StepDefinitions/LanguagesStepDefs.cs#L13